### PR TITLE
Remove tests for input/output summaries

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Orange 3.29 already supports automated summaries but they are not available for older versions of Orange. Since orange3-text supports older versions of Orange, they must stay implemented in widgets but tests are not working for newer versions of Orange.

##### Description of changes
Removing tests for summaries

**Discussion needed: some widgets (e.g. Preprocess) also report on the number of tokens. I suggest implementing a similar summary that is in Preprocess widget to all outputs connected to the corpus.** 

##### Includes
- [ ] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by PrimozGodec** [comment]
Tests should stop failing after https://github.com/biolab/orange3-text/pull/661 is merged

**Review by VesnaT — COMMENTED** [review]
The 'summary' code in widgets should be removed along with the tests. 

A failing test is a good reminder of having useless code in a widget.

**Comment by PrimozGodec** [comment]
@VesnaT until we support Orange 3.28 we cannot remove code for summaries since they are still required for 3.28. I could change tests to test summaries on 3.28 and other versions (not sure if it is worth spending time with it). I can anyway make a test that will start failing when we raise the minimal version to 3.28.

File: orangecontrib/text/widgets/owstatistics.py [hunk 1]
```diff
@@ -598,16 +597,8 @@ def set_data(self, corpus) -> None:
         self.result_dict = {}  # empty computational results when new data
         # reset old output - it also handle case with corpus == None
         self.Outputs.corpus.send(None)
-
-        # summary
         if corpus:
```
Comment (VesnaT) [inline orangecontrib/text/widgets/owstatistics.py:600]: The `if` is no longer relevant.

File: orangecontrib/text/widgets/owwordcloud.py [hunk 2]
```diff
@@ -491,53 +487,6 @@ def commit(self):
...
-                input_numbers += str(len(self.corpus_counter))
-            if self.topic is not None:
-                input_string += f"{self.n_topic_words} words in a topic."
-                input_numbers += (
-                    f"{' | ' if input_numbers else ''}" f"{self.n_topic_words}"
-                )
-            self.info.set_input_summary(input_numbers, input_string)
-
-    def update_output_summary(
-        self, cor_output_len: Optional[int], n_selected: Optional[int]
```
Comment (VesnaT) [inline orangecontrib/text/widgets/owwordcloud.py:519]: There is an unused import of `Optional` among imports.

File: orangecontrib/text/widgets/owdocumentembedding.py [hunk 3]
```diff
@@ -143,32 +143,9 @@ def _setup_layout(self):
...
-                                        .format(len(corpus)))
-
-    def set_output_corpus_summary(self, embeddings, skipped):
-        if embeddings is None and skipped is None:
-            self.info.set_output_summary(self.info.NoOutput)
-        else:
-            successful = len(embeddings) if embeddings else 0
-            unsuccessful = len(skipped) if skipped else 0
-            if unsuccessful > 0:
-                self.Warning.unsuccessful_embeddings()
```
Comment (VesnaT) [inline orangecontrib/text/widgets/owdocumentembedding.py:160]: The 'warning' code should not be removed.

**Comment by PrimozGodec** [comment]
Thank you for spotting those errors. It should be fixed now.

File: orangecontrib/text/corpus.py [hunk 4]
```diff
@@ -657,3 +658,20 @@ def arrays_equal(a, b):
...
+@summarize.register(Corpus)
+def summarize_(corpus: Corpus) -> PartialSummary:
+    """
+    Provides automated input and output summaries for Corpus
+    """
+    table_summary = summarize.dispatch(Table)(corpus)
+    extras = (
+        (
+            f"<br/><nobr>Total tokens: {sum(map(len, corpus.tokens))}, "
+            f"Total types: {len(corpus.dictionary)}</nobr>"
```
Comment (PrimozGodec) [inline orangecontrib/text/corpus.py:672]: In my opinion total types are less understandable. @VesnaT what do you think bout replacing it with `Number unique tokes` or something similar.

**Pull request merged**